test-in-qmlscene.sh

```diff
@@ -13,11 +15,22 @@ done < <(find */ -maxdepth 0 -type d -print0 )
 if ! command -v qmlscene &> /dev/null
 then
     echo "
-$(tput setaf 1)qmlscene could not be found. Install qtcreator from your package manager.$(tput sgr0)"
+$(tput setaf 1)Qmlscene could not be found. Install qtcreator from your package manager.$(tput sgr0)"
     exit
 else
     echo "
-$(tput setaf 2)qmlscene found, proceeding...$(tput sgr0)
+$(tput setaf 2)Qmlscene found, proceeding...$(tput sgr0)
+         "
+fi
+
+if ! command -v magick &> /dev/null
```

Comment:
Could you do the same trick as https://github.com/AsteroidOS/asteroid-wallpapers/pull/8 ?

Reply:
Absolutely!